app/models/article.rb

```diff
@@ -1,29 +1,27 @@
 # frozen_string_literal: true
 
 class Article < ApplicationRecord
+  # this is required for the following function
+  # scope :sorted, -> { order(created_at: :desc) }
```

Comment:
do we need this comment here? Its not clear for me why do we need this from PR description.
"This method scope :sorted, -> { order(created_at: :desc) } will be use in the following function."
And why not to add factories now?)

Reply:
I was using default_scope but the linters wouldn't let me do that. I converted to a regular scope. But I plan to use it in the views - this is another PR. And linters again wouldn't let me do that, because I don't use this scope in this PR. I decided to comment this row :)
And, unfortunately, I don't know how to create factories now and use them :( I'm going to learn about this.

Reply:
i will suggest to create factories in this pr. If you are not going to create factories now, i can approve this pr.

Reply:
I'm still learning how to create factories and use them in RSpec. I'm going to add them in the next pr.